app/api/currentuser/accept_group_invitation.feature

```diff
@@ -23,13 +23,13 @@ Feature: User accepts an invitation to join a group
 
   Scenario: Successfully accept an invitation
     Given I am the user with ID "1"
-    When I send a POST request to "/current-user/group-invitations/11/accept"
```

Comment:
Why changing it? actually for REST "actions", I usually find better to use POST. This is an action with the several impacts, it is not just an update of an attribute by one given as parameter.

Reply:
It's kinda strange to have POST services returning 201 with message "updated" instead of "created". These two services actually update relations, not create new ones.

Reply:
`groupRequestCreate` stays POST

Reply:
okay, now it's fixed